play/src/front/Components/Input/InputLabel.svelte

```diff
@@ -1,21 +1,52 @@
-<script>
-    export let id = "";
-    export let label = "";
-    export let value = "";
+<script lang="ts">
+    export let id: string;
```

Comment:
We could make "id" optional, and generate a random "id" if none is supplied (we still need an id to link the label to the input field)